FileUtility.java

```diff
@@ -0,0 +1,92 @@
+import java.io.*;
+import java.text.SimpleDateFormat;
+
+import javax.swing.*;
+import java.util.*;
+import javax.swing.filechooser.FileNameExtensionFilter;
+import org.apache.poi.ss.usermodel.*;
+import org.apache.poi.xssf.usermodel.*;
+
+public class FileUtility extends ReDoReport {
```

Comment:
The name `ReDoReport` might mean something to you but in general you want it to make sense to someone that has very little to no context with the project. I would rename to something like `HoursReport` or something else generic that serves as a good project name.

Reply:
changed to HoursReport